api/observability/v1/clusterlogforwarder_types.go

```diff
@@ -301,7 +301,7 @@ type ClusterLogForwarderStatus struct {
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
 // +kubebuilder:resource:categories=observability,shortName=clf
```

Comment:
Does the shortname of obsclf also have to be added here?

Reply:
Probably, good catch.

Reply:
It's what I get for "sneaking" in a few other changes.